concordium-consensus/test-runners/execute-chain/Main.hs

```diff
@@ -43,12 +43,12 @@ main = do
     createDirectoryIfMissing True dataDir
     let config ::
             MultiVersionConfiguration
-                (PairGSConfig DiskTreeDiskBlockConfig MemoryTreeMemoryBlockConfig)
+                DiskTreeDiskBlockConfig
```

Comment:
Either revert this or fix the warning from the import of `Paired`.

Reply:
This was committed by mistake.